### PR TITLE
Removing duplicated div

### PR DESCRIPTION
**Pull request by kensherman**

### CODE REVIEW

**Review by Asizon — APPROVED** [review]
Indendation is not ok at all, but aproved

File: src/pages/index.ejs [hunk 1]
```diff
@@ -200,9 +200,7 @@ description: "Betaflight is flight controller software (firmware) used to fly mu
 						</div>
 						<section class="video-area col-lg-6">
 							<div class="overlay overlay-bg"></div>
-							<div class="container">
-								<div class="video-content">
-			
+								<div class="container">
```
Comment (haslinghuis) [inline src/pages/index.ejs:203]:
```suggestion
						<div class="container">			
```

We also need to remove line 207

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: src/pages/index.ejs (hunk 1 above)
Comment (Asizon) [reply src/pages/index.ejs:203]: Yeah good catch

Comment (sugaarK) [reply src/pages/index.ejs:203]: @kensherman  fix this up and we’ll merge it

Comment (kensherman) [reply src/pages/index.ejs:203]: Have run html checker on all files. There's a lot of noise from the esj script tags but think I've got the hmtl ones fixed

**Review by haslinghuis — DISMISSED** [review]

**Review by blckmn — APPROVED** [review]

**Comment by blckmn** [comment]
Needs a rebase

**Review by sugaarK — APPROVED** [review]

**Comment by Asizon** [comment]
Ok, so this PR code breaks again our web page, @kensherman can you please fix it?

**Comment by Asizon** [comment]
image

**Comment by Asizon** [comment]
What is the reason to remove Header Id from code @kensherman ??

**Pull request merged**